dev/frontend/src/lib/query.js

```diff
@@ -84,46 +84,61 @@ export default {
     },
 
     checkin: async (data) => {
+        
+        let events_pb = []
         try {
```

Comment:
do we have to put the pb call in the try catch block when we don't handle anything in catch?